util/warnp.h

```diff
@@ -8,6 +8,8 @@
 #define warn libcperciva_warn
 #define warnx libcperciva_warnx
 
+#define WARNP_SYSLOG_MAX_LINE 255
```

Comment:
Let's make this 4095.  I can imagine logging warnings which are more than 255 characters.

Reply:
done